src/OpenLoco/src/Ui/Window.h

```diff
@@ -267,9 +229,16 @@ namespace OpenLoco::Ui
         AdvancedColour colours[enumValue(WindowColour::count)]; // 0x886
         int16_t var_88A;
         int16_t var_88C;
+        sfl::small_vector<Widget, 16> widgets;
```

Comment:
How does small vector work?

Reply:
https://github.com/slavenf/sfl-library/blob/master/doc%2Fsmall_vector.md

Personally I don't see why we need to pull in another library for some unnecessarily-specialized vector type.

Reply:
The main advantage of the sfl library is static allocation over heap allocation. 

We've already included the library for the window list, btw.

Reply:
`unnecessarily-specialized vector type`, I don't see it that way, we also use static_vector from it and I plan to make more use of the other ones, I integrated this library in other personal projects and it works really well.